lib/rebaser-server/src/serial_dvu_task.rs

```diff
@@ -65,10 +65,6 @@ impl SerialDvuTask {
     pub(crate) async fn try_run(self) -> Result<Shutdown> {
         metric!(counter.serial_dvu_task.change_set_in_progress = 1);
 
-        // Attempt to run an initial dvu in case there are processed requests that haven't yet been
-        // finished with a dvu run
-        self.maybe_run_initial_dvu().await?;
```

Comment:
@fnichol hey! Two things here

1. @britmyerss and I looked at this and we believe it might not have been doing anything.
2. We believe it may also be a potential culprit for the `PINGA_JOBS` poison pill issues, though there is not hard evidence yet other than static analysis, talking it through and timeline.

We believe this is safe to remove and it is not essential for use in the rebaser, but I wanted to check with you for review!

Reply:
I rolled this change over and over in my head and landed on: "yeah, that's going to work for us" 😄